elytron/WFLY-16532-additional-scope-for-auth-request.adoc

```diff
@@ -0,0 +1,129 @@
+== Adding the ability to configure additional scope for authentication request
+:author:            Prarthona Paul
+:email:             prpaul@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+OpenID Connect is an authentication mechanism that builds on OAuth 2.0 
+and allows a user to login to a web application using credentials established 
+by an OpenID provider. 
+Currently, when sending an authentication request to the OpenID provider, one 
+of the required parameters with the authentication flow is "scope". However, for 
+now, that value is hardcoded as just "openid". 
+
+The specifications indicate that there are other scope values which may be included in 
+the authentication request. This new feature adds the ability to configure the `scope` attribute
+of the `elytron-oidc-client` subsystem, so that those additional parameters can be specified when 
+configuring the server. 
+
+The feature will allow the user to configure the server in two ways: 
+
+* In an application's oidc.json file 
```

Comment:
This configuration file is included in the deployment (not in the server configuration).

So the references above to "configuring the server" and "configure the server" should be updated to reflect this.